draft-ietf-idr-linklocal-capability.xml

```diff
@@ -50,14 +50,26 @@
     <area>Routing</area>
     <workgroup>Interdomain Routing</workgroup>
     <abstract>
-      <t>BGP <xref target="RFC4271" format="default"/>, was originally designed
-to provide reachability between domains and between the edges of a domain.
-As such, BGP assumes the next hop towards any reachable destination may
-not reside on the advertising speaker, but rather may either be through
-a router connected to the same subnet as the speaker, or through a router
-only reachable by traversing multiple hops through the network. Because
-of this, as per <xref target="RFC4291" format="default"/> - BGP does not recognize  IPv6 link-local addresses,
-as a valid next hop for the forwarding purposes.</t>
+      <t>
+        BGP <xref target="RFC4271" format="default"/>, was originally designed
+        to provide reachability between domains and between the edges of a domain.
+      </t>
+      <t>
+        To support IPv6 reachability, BGP relies heavily on its Multiprotocol Extensions
+        as defined in <xref target="RFC2545" format="default"/>, which is crucial for
+        enabling BGP-4 to advertise IPv6 routes alongside IPv4. This extension not only
+        permits the exchange of IPv6 routing information but also establishes the structure
+        for IPv6 next-hop handling within BGP updates. As such, <xref target="RFC2545" format="default"/>
+        is fundamental to BGP's operation in IPv6 networks, ensuring that IPv6 prefixes
+        are propagated consistently and efficiently across domains.
+        As such, BGP assumes the next hop towards any reachable destination may
+        not reside on the advertising speaker, but rather may either be through
+        a router connected to the same subnet as the speaker, or through a router
+        only reachable by traversing multiple hops through the network. Because
+        of this, as per <xref target="RFC4291" format="default"/> - BGP does not
+        recognize  IPv6 link-local addresses, as a valid next hop for the forwarding
+        purposes.
```

Comment:
I don't think this text draws a clear conclusion for its RFC 4291 relationship.  BGP does permit peering and nexthops with directly attached systems along with those that may not be directly attached - primarily in an ibgp sense.  IPv6 didn't change that.

What RFC 2545 added as the ability to attach a global address, and when the system was directly attached, both a global and a link local.

What is being corrected in this draft is the encoding when the system is directly attached and only has link-local addressing for its BGP nexthop.

Reply:
Updated, how about now?